### PR TITLE
added setup.py file

### PR DESCRIPTION
**Pull request by bearecinos**
I added the `setup.py` file to try to install things via pip.

Let me know if we need to add something else.

Cheers 
Bea

### CODE REVIEW

**Comment by bearecinos** [comment]
All pytest seem to pass in the mamba workflow:
https://github.com/bearecinos/github_conda_workflow/runs/7301898148?check_suite_focus=true

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+import setuptools
+if __name__ == "__main__":
+    setuptools.setup(
+    name="fenics_ice",
+    description="A finite element model framework "
+                "that quantifies the initialization uncertainty for "
+                "time-dependent ice sheet models.",
+    url="https://github.com/EdiGlacUQ/fenics_ice.git",
+    license="GNU LGPL version 3",
```
Comment (jrmaddison) [inline setup.py:10]: `packages=["fenics_ice"],`

**Review by jrmaddison — CHANGES_REQUESTED** [review]
Should be linted with flake8.

File: setup.py (hunk 1 above)
Comment (bearecinos) [reply setup.py:10]: commit [2fcbd5](https://github.com/EdiGlacUQ/fenics_ice/pull/98/commits/2fcbd558c2b871b61e1dae5f9bfd9f604f432ff2)

**Comment by bearecinos** [comment]
@jrmaddison  managed to add flake8 to pycharm ... should give me indentation corrections

**Review by jrmaddison — APPROVED** [review]

**Pull request merged**